### ISSUE TITLE
RGB image question

### ISSUE THREAD

**Issue by missswiftie** [StanfordVL/BEHAVIOR-1K#1956]
rgb = self.camera._get_obs()[0]['rgb']
rgb = rgb.detach().cpu().numpy()
rgb = cv2.cvtColor(rgb, cv2.COLOR_BGR2RGB)
When I run the code above, I get the rgb image which is red:

<img width="615" height="405" alt="Image" src="https://github.com/user-attachments/assets/17ae4c2f-e6ba-41bc-9c17-5d791732012e" />

I tried all the channel permutation but failed to get the regular rgb image.

Thank you for answering!

**Comment by cgokmen** [comment]
The image is already RGB, there's no need to convert it from BGR. Try `self.camera.get_obs()[0]["rgb"].cpu().numpy()[:, :, 3].astype(np.uint8)` - there might be additional channels there or a bad dtype that are causing this confusion
